app/src/interfaces/_system/system-fields/system-fields.vue

```diff
@@ -100,6 +107,15 @@ const fields = computed<(Field & { key: string })[]>({
 
 const { t } = useI18n();
 
+function selectAll() {
+	const newArray = fieldsStore.getFieldsForCollection(props.collectionName)?.map((field) => field.field);
```

Comment:
```suggestion
	const newArray = fieldsStore.getFieldsForCollection(props.collectionName).map((field) => field.field);
```

The return type for `fieldsStore.getFieldsForCollection()` is `Field[]`, is the optional chaining required?